ksqldb-rest-client/src/main/java/io/confluent/ksql/rest/client/KsqlTarget.java

```diff
@@ -73,28 +73,31 @@
   private final SocketAddress socketAddress;
   private final LocalProperties localProperties;
   private final Optional<String> authHeader;
+  private final String host;
 
   KsqlTarget(
       final HttpClient httpClient,
       final SocketAddress socketAddress,
       final LocalProperties localProperties,
-      final Optional<String> authHeader
+      final Optional<String> authHeader,
+      final String host
```

Comment:
why are we passing in `host` here? I'm assuming this is different to `socketAddress.getHost`.  This is really counter-intuitive and the kind of thing that will trip people up later when they make changes to this class.

Is there any reason by you can't just change the calling code to pass in the _right_ `socketAdrress`?

Reply:
Looks good.